Header header

float64 x # +x: East
float64 y # +y: South
float64 z # +z: Upwards
float64 theta # in radians; 0: East, pi: North
float64 linear_velocity
float64 angular_velocity # in radians / s
float64 linear_velocity_z
